cypress/support/pages/tasks/pageTasks.js

```diff
@@ -298,12 +315,9 @@ class Tasks {
   }
 
   clickDownloadFileInSubmission (fileName) {
-    cy.get(Tasks.#submissionsSection)
-      .find('a')
-      .should('contain', fileName)
-      .parent()
-      .find('span')
-      .find('a')
+    cy.get(`[data-file-name="${fileName}"]`)
```

Comment:
I would suggest not do like that but something like 
``` 
let randomSelector = '[data-file-name='
let newRandomSelector = randomSelector.concat(fileName, ']')
cy.get(newRandomSelector)
...
```

Reply:
Is this more performant?

Reply:
-> https://ticketsystem.dbildungscloud.de/browse/BC-2936